api/app/signals/apps/signals/serializers.py

```diff
@@ -30,7 +30,17 @@
 log = logging.getLogger(__name__)
 
 
-class LocationModelSerializer(serializers.ModelSerializer):
+class NearAmsterdamValidatorMixin:
+    def validate_geometrie(self, value):
+        fail_msg = 'Location coordinates not anywhere near Amsterdam. (in WGS84)'
+
+        if (not 1 < value.coords[0] < 7) or (not 50 < value.coords[1] < 55):
```

Comment:
Suggestion, can you make this conditions more descriptive. Something like `lat_not_in_adam_area` and `lon_not_in_adam_area` or something better .. (?)

Reply:
Fixed by @tcoenen